teams/teams-ps/teams/Get-CsPolicyPackage.md

```diff
@@ -40,7 +40,23 @@ Returns all policy packages available on the tenant.
 PS C:\> Get-CsPolicyPackage -Identity Education_PrimaryStudent
 ```
 
-Returns the definition of the Education_PrimaryStudent policy package, including the policies applied with the package.
+Returns only the Education_PrimaryStudent policy package.
+
+### Example 3
+```powershell
+PS C:\> $a = Get-CsPolicyPackage -Identity Education_PrimaryStudent
+PS C:\> $a.Policies.AdditionalProperties
```

Comment:
```suggestion
PS C:\> $a.Policies
```
I think we should remove ".AdditionalProperties":
![image](https://user-images.githubusercontent.com/33589238/99400188-6ef2b400-28c5-11eb-900e-6e24fff78d42.png)

Reply:
Agreed, also in the upcoming release the AdditionalProperties will need to be specified. I'll put a comment in there with a version placeholder and I'll update this PR once I know what the version number is going to be